cmd/panacead/cmd/root.go

```diff
@@ -115,6 +115,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		AddGenesisWasmMsgCmd(app.DefaultNodeHome),
 		AddGenesisOracleCmd(app.DefaultNodeHome),
 		EncryptDataCmd(app.DefaultNodeHome),
+		DecryptData(app.DefaultNodeHome),
```

Comment:
How about we change to `DecryptData` as other cmd function name?

Reply:
fixed: https://github.com/medibloc/panacea-core/pull/545/commits/b432c3840b8748e98751b9b541387e82d7aaf4c4